src/views/labs/LabView/LabView.vue

```diff
@@ -11,6 +11,42 @@
     </b-tabs>
     <LabRound v-else v-for="round in closedRounds" :key="round.round" :round="round" closed="true" />
     <LabConclusion v-if="lab.conclusion" :conclusion="lab.conclusion" style="margin-bottom: 52.5px;" />
+    <p></p>
+    <p></p>
```

Comment:
```suggestion
```
The margins of the h3 are already taking care of this

Reply:
Is there an easier way to space out elements? The reason this was included was to add more space between the top of the puzzle card, as the h3 header isn't spaced out enough.

Reply:
I'm pretty sure just using the h3 makes it just as spaced out as all the other sections. When I looked at it, either adding additional spacing makes it arbitrarily different or actually doesn't change it at all (because the only reason the p elements have space is because they have margins, and due to the way margins work, margins on sibling elements get collapsed rather than adding on to each other) - I can't remember offhand which one it was. That said - the way you would do it is by increasing the margin-top on the h3.